### ISSUE TITLE
Fail to install Vundle on Unix without Internet Connection

### ISSUE THREAD

**Issue by qizy09** [VundleVim/Vundle.vim#737]
Couldn't install it on unix server, which is only available with Intranet. I copied the git Repo to the directory ~/.vim/bundle/Vundle.vim

I tried to install with "vim +PluginInstall +qall", with error below.
![image](https://cloud.githubusercontent.com/assets/6061320/16242161/f8e4ca7e-37a5-11e6-8043-091d7bc85acb.png)

Then when I run vim, I would receive
![image](https://cloud.githubusercontent.com/assets/6061320/16242090/a14312bc-37a5-11e6-968e-3cdb0421e12d.png)

**Comment by nfischer** [comment]
Vundle uses the internet to download your plugins. It'll never be able to run `: PluginInstall` without a connection

**Comment by ryanoasis** [comment]
Yeah it doesn't make perfect sense to use vundle without an internet connection but this also doesn't fully explain the errors.

@qizy09 do you still have these errors? Which version of Vim?

**Comment by nfischer** [comment]
#102 also has the `missing braces` error message. It may be related.
